op/model.py

```diff
@@ -252,18 +252,12 @@ def fetch(self, name):
         """Fetch the resource from the controller or store.
 
         If successfully fetched, this returns a Path object to where the resource is stored
-        on disk, otherwise it returns None.
+        on disk, otherwise it raises a CalledProcessError.
```

Comment:
We definitely don't want methods returning CalledProcessError.  I suggest we transform it to ModelError instead, which allows people to catch error more generically, and allows us to specialize errors as we move forward without breaking existing logic.

@dshcherb Wanna work on this?

Reply:
#60